crates/sui-types/src/messages_consensus.rs

```diff
@@ -273,6 +275,8 @@ pub enum ConsensusTransactionKind {
     RandomnessDkgConfirmation(AuthorityName, Vec<u8>),
 
     CapabilityNotificationV2(AuthorityCapabilitiesV2),
+
+    UserTransaction(Box<Transaction>),
```

Comment:
I would think this should be a `VerifiedTransaction`, to make sure we remember to verify the transaction before submitting it to consensus.